app/pug/common.pug

```diff
@@ -1,49 +1,17 @@
 
+block pugscripts
+  - function isAuthenticated()     { return !!(user); }
+  - function user_email(aUser)     { if (isAuthenticated()) return ((aUser.emails || [])[0] || {}).value || ""; else return "[Please login to see!]" };
```

Comment:
I'll actually simplify the return statement to: `return (user.emails[0].value);` .  The current syntax assumes that a user might not have an associated email address, but with the new Auth0 setup this is never the case any more (since we now require users to have at least one).  Will still add a comment to describe the function anyway.